frontend/projects/upgrade/src/app/features/dashboard/home/components/modal/import-experiment/import-experiment.component.ts

```diff
@@ -511,7 +509,7 @@ export class ImportExperimentComponent implements OnInit {
     result = this.deduceConditionPayload(result);
```

Comment:
All the deduce function is mutating the result object. So you don't need to return anything.

Reply:
code updated